unirecfilter/lib/functions.c

```diff
@@ -167,6 +173,35 @@ struct ast *newExpressionFP(char *column, char *cmp, double number)
    return (struct ast *) newast;
 }
 
+struct ast *newExpressionDateTime(char *column, char *cmp, char *datetime)
+{
+   struct expression_datetime *newast = (struct expression_datetime *) malloc(sizeof(struct expression_datetime));
```

Comment:
Missing check whether malloc succeeded. I noticed it missing in many places in NEMEA but we should start improving it at one point :) Even if it will decrease our Codecov.